bears/cmake/CMakeLintBear.py

```diff
@@ -21,11 +23,19 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         max_line_length: int=79,
```

Comment:
before your change, the default line length check performed by `cmakelint` was infinity.  It wasnt checking line length.

You are introducing that feature, and it should not be defaulting to 79.  That will mean anyone who was using this bear and had lines longer than 79 chars will have their CI break when they upgrade to the next version of coala.  That isnt nice.